modules/hostap/CMakeLists.txt

```diff
@@ -58,20 +58,20 @@ zephyr_library_compile_definitions_ifdef(CONFIG_WIFI_NM_WPA_SUPPLICANT_NO_DEBUG
   CONFIG_NO_STDOUT_DEBUG
 )
 
-zephyr_library_compile_definitions_ifdef(CONFIG_WIFI_NM_WPA_SUPPLICANT_ROBUST_AV
-  CONFIG_ROBUST_AV
+zephyr_library_compile_definitions_ifndef(CONFIG_WIFI_NM_WPA_SUPPLICANT_ROBUST_AV
+  CONFIG_NO_ROBUST_AV
 )
 
-zephyr_library_compile_definitions_ifdef(CONFIG_WIFI_NM_WPA_SUPPLICANT_WMM_AC
-  CONFIG_WMM_AC
+zephyr_library_compile_definitions_ifndef(CONFIG_WIFI_NM_WPA_SUPPLICANT_WMM_AC
+CONFIG_NO_WMM_AC
 )
 
-zephyr_library_compile_definitions_ifdef(CONFIG_WIFI_NM_WPA_SUPPLICANT_RRM
-  CONFIG_RRM
+zephyr_library_compile_definitions_ifndef(CONFIG_WIFI_NM_WPA_SUPPLICANT_RRM
+CONFIG_NO_RRM
 )
 
 zephyr_library_compile_definitions_ifdef(CONFIG_WIFI_NM_WPA_SUPPLICANT_ROBUST_AV
```

Comment:
Should this be `ifndef`?

Reply:
Yes, but this was a duplicate definition, the original still had `ifndef` I will just remove it.